exec/bigmachine.go

```diff
@@ -52,7 +52,7 @@ const (
 )
 
 // RetryPolicy is the default retry policy used for machine calls.
-var retryPolicy = retry.Backoff(time.Second, 5*time.Second, 1.5)
+var retryPolicy = retry.MaxTries(retry.Backoff(5*time.Second, 60*time.Second, 2), 5)
```

Comment:
What's the reasoning for increasing the minimum retry time?

Reply:
This retry policy is used by the `retryReader` which is used for both worker-to-worker reading during evaluation and the final result scan reading.  Now that the final result scan reading can now trigger evaluation, which transitively can potentially bring up machines to do that evaluation, I thought I'd lengthen it to give some time for machines to come up if necessary.  This might make retries for worker-to-worker reads a bit slower, but that happens pretty rarely, and I don't think it will make much of a difference in the grand scheme of things.  Note that I did not do any empirical testing -- just thought experiments.  Let me know if you think that is unreasonable.